diff --git a/include/bit7z/bitwindows.hpp b/include/bit7z/bitwindows.hpp
index a86edca..5849b95 100644
--- a/include/bit7z/bitwindows.hpp
+++ b/include/bit7z/bitwindows.hpp
@@ -18,7 +18,7 @@
 #ifndef NOMINMAX
 #define NOMINMAX
 #endif
-#include <Windows.h>
+#include <windows.h>
 #include <propidl.h>
 #else
 
diff --git a/src/internal/stringutil.cpp b/src/internal/stringutil.cpp
index 9a0a3f4..2a6dfc3 100644
--- a/src/internal/stringutil.cpp
+++ b/src/internal/stringutil.cpp
@@ -15,7 +15,7 @@
 #include "internal/stringutil.hpp"
 
 #ifdef _WIN32
-#include <Windows.h>
+#include <windows.h>
 #ifdef BIT7Z_USE_SYSTEM_CODEPAGE
 #define CODEPAGE CP_ACP
 #define CODEPAGE_WC_FLAGS WC_NO_BEST_FIT_CHARS
